remove-vowels-2/mutation_3.py

```diff
@@ -0,0 +1,11 @@
+def remove_vowels(word: str, y_is_a_vowel: bool) -> str:
+    '''
+    Omitting to check for spaces in the word
+    '''
+    vowels = "aeiouAEIOU"
+    vowels += "yY" if y_is_a_vowel else ""
+    result = ""
+    for letter in word: # or letter == " "     is missing
```

Comment:
wouldn't for `for letter in word` still check for empty spaces as they are technically elements of the list word?